docs/transaction-building.md

```diff
@@ -0,0 +1,264 @@
+# Transaction Building
+
+An Ergo transaction is a way to interact with the blockchain. A transaction takes Input boxes from the blockchain, describes how each must be spent, and generates new Output boxes. These Output boxes then serve as Input boxes for a later transaction.
+
+- Each and every Ergo transaction is an **atomic operation**, which means it can not be partially executed.
+
+- Each transaction must contain **one or more** Input boxes, **one or more** Output boxes and **zero or more** Data-Input boxes.
+
+:::warning
+**Work-in-Progress**: Non-reviewed text. You may find numerous writing errors throughout this guide.
+:::
+
+## What is a box?
+
+Shortly a box is an **Extended Unspent Transaction Output (eUTxO)**, which extends the Bitcoin's **Unspent Transaction Output (UTxO)** data structure with tokens, registers, data, and smart contracts.
+
+An Ergo Transaction is nothing more than a set of boxes.
```

Comment:
```suggestion
An Ergo Transaction is nothing more than changing boxes, inputs into outputs.
```

Reply:
The transaction is making a change. I feel this is important to include. An address can have a set of boxes, but it is not a transaction. The defining characteristic of a transaction is it changes the blockchain, which means changing boxes. Maybe pedantic, but it seems clearer to me. Maybe there's another way to say it too?

Reply:
Yeah, good point! When I said "a set of boxes" I was referring to the transaction's data structure. But your approach is better as it covers the whole concept of a transaction, not only the data structure.